02_bubble/kaizhu/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(sc []int) []int {
+	s := make([]int, len(sc))
+	copy(s, sc)
+	for i := 0; i < len(s)-1; i++ {
```

Comment:
The len(s) could be called outside the for loop by assigning the value to a variable, otherwise, this function will be called every loop.